Simulator/Recorders/XmlRecorder.cpp

```diff
@@ -76,27 +76,33 @@ void XmlRecorder::term()
 /// @param[in] neurons    The entire list of neurons.
 void XmlRecorder::compileHistories(AllVertices &vertices)
 {
-   if (singleNeuronEvents_ == nullptr) {
-      return;
+   for (int iNeuron = 0; iNeuron < variableTable_.size(); iNeuron++) {
```

Comment:
Change all references to the word "neuron" to "variable", I think.

Reply:
This is a nested for loop to iterate the variableTable_ . Would I use rowIndex and columnIndex for each loop?